pulp_smash/tests/pulp3/utils.py

```diff
@@ -186,8 +187,25 @@ def publish_repo(cfg, publisher, repo, version_href=None):
     call_report = client.post(urljoin(publisher['_href'], 'publish/'), body)
     # As of this writing, Pulp 3 only returns one task. If Pulp 3 starts
     # returning multiple tasks, this may need to be re-written.
-    last_task = next(api.poll_spawned_tasks(cfg, call_report))
-    return client.get(last_task['created_resources'][0])
+    verify_number_tasks(len(call_report))
+    tasks = next(api.poll_spawned_tasks(cfg, call_report))
+    return client.get(tasks['created_resources'][0])
+
+
+def verify_number_tasks(number_tasks):
```

Comment:
It'd be very useful to print the contents of a malformed call report within this function.